bom/pom.xml

```diff
@@ -10,10 +10,10 @@
 	<packaging>pom</packaging>
 	<properties>
 		<migrator.version>2.56.1</migrator.version>
-		<client.version>5.16.0</client.version>
+		<client.version>5.19.0</client.version>
 		<core.version>2.17.0</core.version>
-		<mongo.version>1.34.0</mongo.version>
-		<rest.version>2.27.0</rest.version>
+		<mongo.version>1.38.0</mongo.version>
+		<rest.version>2.30</rest.version>
```

Comment:
This should be 2.30.0 (I know I had it wrong in the other MR_

Reply:
Done